hexrd/ui/resources/calibration/default_tardis_config.yml

```diff
@@ -9,8 +9,8 @@ detectors:
       size: [0.1, 0.1]
     buffer: [0.2, 0.2]
     transform:
-      tilt: [0.0, 0.0, 0.0]
-      translation: [0.0, 0.0, 0.0]
+      tilt: [-1.57079633e+00, 0.0, 0.0]
```

Comment:
Same here. Can this be `-1.57`, or does it need to be very precise?

Reply:
That number is `pi/2`. I think having it precise is good.